### ISSUE TITLE
--site-list option is not working

### ISSUE THREAD

**Issue by ycl6** [genome/bam-readcount#44]
Hi,

I git clone and build from the latest source file. I tried the following command to limit the search to a collection of sites, but bam-readcount simply stops without output. 

```
./bam-readcount -w 0 -f GRCh37.primary_assembly.genome.fa --site-list loci.txt Aligned.sortedByCoord.out.bam
```
```
$ head loci.txt
chr1:13418-13418
chr1:13649-13649
chr1:324822-324822
chr1:897462-897462
chr1:1580738-1580738
```

**Comment by ernfrid** [comment]
Does that version of the reference include chr prefixes for all the chromosomes?

`bam-readcount` requires that the chromosome names match those in your reference sequence exactly. Usually when no output is returned this is why.

**Comment by ycl6** [comment]
@ernfrid Yes, the reference does have the same chromosome prefix as the bam file. `bam-readcount` returns the result in single-region mode, but not when a file is given.

**Comment by ernfrid** [comment]
Ah. I think I see the issue. The format of the text file should be tab-separated values. From the README:

> The list of regions should be formatted as chromosome start and end. Each field should be tab separated and coordinates should be 1-based.

I think that if you change your file to use tabs then this should resolve the issue. Try:

```chr1    13418   13418
chr1    13649   13649
chr1    324822  324822
chr1    897462  897462
chr1    1580738 1580738

**Comment by romunov** [comment]
IMO information on formatting of the list should be included in the help itself, too, not README only.
